### ISSUE TITLE
oryp7 GPU temperature reports incorrectly

### ISSUE THREAD

**Issue by jackpot51** [system76/firmware-open#189]
As reported in https://www.reddit.com/r/System76/comments/mdexdx/hybrid_graphics_no_spinning_gpu_fan/

I have confirmed that if I toggle the GPU power on and off repeatedly, sometimes the GPU gets into a state where it reports back 0 temperature when it is still on. This only appears to happen on the oryp7, not the previous oryp6 model.

**Comment by jackpot51** [comment]
The easiest way to reproduce this:

- Launch Steam
- Close the Steam window
- Wait for GPU temp to go to zero
- Open Steam Library, and potentially launch a game

**Comment by lucserre** [comment]
This may or may not also be a problem, but I think even in the 'integrated graphics' mode, the right side fan doesn't turn on at all. Since the heat pipes are linked together on both CPU and GPU, I believe it might be a better idea to have both fans turn on at a lower speed rather than only one fan at a high (louder) speed.  Thanks for reading.

**Comment by curiousercreative** [comment]
@lucserre yep, that update to fans should be coming soon: https://github.com/system76/ec/pull/177

**Comment by MilesBHuff** [comment]
@curiousercreative [found](https://github.com/system76/ec/pull/190#issuecomment-834724959) that GPU temps are always broken in `compute` mode on a galp5.
I was able to [confirm](https://github.com/system76/ec/pull/190#issuecomment-834750085) this is also the case on an oryp7:
![image](https://user-images.githubusercontent.com/6566982/117504335-48e2ce00-af50-11eb-99ba-5057430bf276.png)

**Comment by Raikiri** [comment]
May or may not be related, but LibreHardwareMonitor (windows) reports incorrectly the temperature of my second SSD. Both SSD's are almost identical (Samsung EVO 960 and 970, i think), but the second one always reports 0C.

**Comment by crawfxrd** [comment]
Needs repro on latest release.
